jupyter_book/commands/__init__.py

```diff
@@ -399,3 +337,103 @@ def init(path, kernel):
     """
     for ipath in path:
         init_myst_file(ipath, kernel, verbose=True)
+
+
+# utility functions
+def builder_specific_actions(exc, builder, output_path, cmd_type, page_name=None):
```

Comment:
please document this function so it's easier for others to maintain

Reply:
@choldgraf thanks, I will document it